config/initializers/paperclip.rb

```diff
@@ -1,2 +1,11 @@
 Paperclip::Attachment.default_options[:url] = "/system/:class/:hash.:extension"
 Paperclip::Attachment.default_options[:hash_secret] =  Rails.application.secrets.paperclip_hash_secret
+
+require 'paperclip/media_type_spoof_detector'
+module Paperclip
+  class MediaTypeSpoofDetector
+    def spoofed?
+      false
```

Comment:
A pity spoofing detection cannot be disabled, it seems it's one of the most buggy features of paperclip.